### DISCUSSION TITLE
Increase the time needed for welding fuel to explode in a fire

### DISCUSSION THREAD

**Discussion by Krzeszny** [FakeFishGames/Barotrauma#5234] [Ideas and suggestions]
Right now if you literally touch a flame, all the welding fuel on you and people nearby explodes.
I'd say min. 10 seconds but that's also very quick.

**Comment by ekqvist** [comment]
We have made some adjustments to this in the latest Unstable builds. Now, when a tank explodes, the tank explosion shouldn't automatically explode all other nearby tanks anymore. The resulting fire can still do this, but this also has a delay to it so the fire will first slowly "consume" the tanks before they will explode. I hope this will help with some of the frustration.

**Comment by Hazmat616** [comment]
Can welding fuel in Flamers get extra time or protection? It's kind of weird to be burning stuff and then suddenly you explode.
